patchwork/views/patch.py

```diff
@@ -136,6 +136,13 @@ def patch_detail(request, project_id, msgid):
         related_same_project = []
         related_different_project = []
 
+    context['next_submission'] = None
+    patch_series = patch.series.patches.all()
+    num_patches = patch.series.patches.count()
```

Comment:
This is a very expensive query. We're not currently joining on `.series` or `.series.patches`, which means we're lazy loading a lot of this. I see our total queries on the page go from 20 to 49 (you can see this yourself using django-debug-toolbar, which is available on the top right of the viewport).

![image](https://github.com/getpatchwork/patchwork/assets/1690835/f45908b1-e32c-409e-8332-a79b242e0a6c)

This needs to be addressed. Given we're already loading some series information to populate the hidden `Series` pane, I think this should be relatively easy to address.